proofreading of Zylevels chapter

tariqkurd-repo opened the pull request:
I've made LR.Y refer to LY and SC.Y refer to SY so that they don't need to both be listed in the Zylevels chaper
AMOSWAP.Y was already done that way

And other very minor consistency and readability fixes

arichardson reviewed (approved):
LGTM with Wes' suggestions

tariqkurd-repo commented:
Fair point, I'll do that on monday

tariqkurd-repo commented:
I've put the instruction pages changes here: https://github.com/riscv/riscv-cheri/pull/747
and the cap tag changes here: https://github.com/riscv/riscv-cheri/pull/746

hoefully that addresses the concerns
